app/src/main/java/com/example/dell/helloandroid/DataBase/Contact.java

```diff
@@ -0,0 +1,58 @@
+package com.example.dell.helloandroid.DataBase;
+
+/**
+ * Created by Omer Majeed on 4/12/2016.
+ */
+public class Contact {
+
+    //private variables
+    int _id;
+    String _name;
+    String _phone_number;
+
+    // Empty constructor
```

Comment:
Comments to be used will be of the format /* */.
@OmerMajeed Kindly remove all the // comments to /* */.